tests/test_imports.py

```diff
@@ -0,0 +1,93 @@
+"""Import tests."""
+
+import importlib
+import os
+import pkgutil
+import subprocess
+import sys
+from collections.abc import Sequence
+
+import pytest
+from pytest import param
+
+pytestmark = pytest.mark.skipif(
+    os.environ.get("BAYBE_TEST_ENV") != "FULLTEST",
+    reason="Only possible in FULLTEST environment.",
+)
+
+_EAGER_LOADING_EXIT_CODE = 42
+
+
+def find_modules() -> list[str]:
+    """Return all BayBE module names."""
+    package = importlib.import_module("baybe")
+    return [
+        name
+        for _, name, _ in pkgutil.walk_packages(
+            path=package.__path__, prefix=package.__name__ + "."
+        )
+    ]
+
+
+def make_import_check(modules: Sequence[str], target: str) -> str:
+    """Create code that tests if importing the given modules also imports the target.
+
+    Args:
+        modules: The modules to be imported by the created code.
+        target: The target module whose presence is to be checked after the import.
+
+    Returns:
+        str: Code that signals the presence of the target via a non-zero exit code.
```

Comment:
The explicit type hint is added automatically, please remove.

Reply:
Thanks, that part was auto-generated and I forgot to delete it! Done